cypress/e2e/auth/auth.cy.js

```diff
@@ -0,0 +1,138 @@
+const { getRandomEmail, getRandomString } = require("../../../utilites/helper");
+const loginPagePO = require("../../page_objects/login_PO");
+
+describe("authentication module", function () {
+  beforeEach(function () {
+    loginPagePO.visitLoginPage();
+  });
+  it("URL should be https://frontendbootcamp.proshore.eu/accounts/login", function () {
+    cy.url().should(
+      "eq",
+      "https://frontendbootcamp.proshore.eu/accounts/login"
+    );
+  });
+  it("verify valid login", function () {
+    cy.login();
+  });
+
+  it(" should not be submit with empty details", function () {
+    loginPagePO.clickLogin();
```

Comment:
the name of the function is not descritive enough, the clickLogin() can rather be clickOnLoginButton()